draftjs_exporter/options.py

```diff
@@ -22,13 +25,19 @@ def __repr__(self):
         return str(self)
 
     def __eq__(self, other):
-        return self.__dict__ == other.__dict__
+        """
+        Equality used in test code only, not to be relied on for the exporter.
```

Comment:
I think that’s ok. It would be fine to use this in the exporter as well, it’s just that it’s not at the moment. In test code it does save quite a lot of boilerplate.